go/saltpack/doc.go

```diff
@@ -0,0 +1,43 @@
+/*
+Package saltpack is an implementation of the Saltpack message format. Saltpack
+is a very light wrapper around Dan Berstein's famous NaCl library. It adds
```

Comment:
I'm not sure I agree with "light wrapper around NaCl". Maybe something more like a "combination of NaCl and MessagePack into a message format for (some) humans."